HelloWorld/HelloWorld.cs

```diff
@@ -7,6 +7,14 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
```

Comment:
The assignment had 10 parts:
https://csharp.austincodingacademy.com/_book/variablesoperatorslesson.html